client/influxdb.go

```diff
@@ -73,18 +74,22 @@ func ParseConnectionString(path string, ssl bool) (url.URL, error) {
 // Username/Password are optional.  They will be passed via basic auth if provided.
 // UserAgent: If not provided, will default "InfluxDBClient",
 // Timeout: If not provided, will default to 0 (no timeout)
+// InsecureSkipVerify: controls whether a client verifies the
```

Comment:
I know naming is hard. I think a terser option would be best. How about:

`VerifySkip`

Simple as that?